### ISSUE TITLE
Mouse hover is not working without chart dimensions

### ISSUE THREAD

**Issue by re-ynd** [entronad/graphic#37]
When I try to show a `Chart` without specifying dimensions for its parent, the mouse hover and possibly other gesture functions are not working. As soon as I specify the `width` and `height` for the the container which encapsulated the `Chart`, the gestures work. Here is an example code which you can use to recreate the bug.

If you run the exact code in a browser, you can see the crosshairs. But if you remove `width: 600` and/or `height: 400` from the code below, the hover stops working and I cannot see the crosshairs.

```dart
import 'package:flutter/gestures.dart';
import 'package:flutter/material.dart';
import 'package:graphic/graphic.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  const MyHomePage({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Container(
          // Commenting out the next two lines is causing the problem
          width: 600,
          height: 400,
          child: Chart(
            data: [
              {'year': DateTime(2010), 'price': 10},
              {'year': DateTime(2011), 'price': 8},
              {'year': DateTime(2012), 'price': 12},
              {'year': DateTime(2013), 'price': 16},
              {'year': DateTime(2014), 'price': 9},
              {'year': DateTime(2015), 'price': 17},
              {'year': DateTime(2016), 'price': 13},
            ],
            variables: {
              'year': Variable(
                accessor: (Map map) => map['year'] as DateTime,
              ),
              'price': Variable(
                accessor: (Map map) => map['price'] as int,
              ),
            },
            elements: [LineElement()],
            axes: [
              Defaults.horizontalAxis,
              Defaults.verticalAxis,
            ],
            selections: {
              'tooltipMouse': PointSelection(
                on: {GestureType.hover},
                devices: {PointerDeviceKind.mouse},
              ),
            },
            crosshair: CrosshairGuide(),
          ),
        ),
      ),
    );
  }
}
```

Any help is much appreciated. Thanks!

**Comment by entronad** [comment]
This is bug, when parent container has no indicated size, the chart will not inflate (Though painting is not affected).
It has been fixed in v0.7.0, it will always inflate the parent container now. Please upgrade and have a try.

**Comment by re-ynd** [comment]
Great! It works well with v0.7.0. Thanks!
